.github/workflows/push-trigger.yml

```diff
@@ -85,7 +85,7 @@ jobs:
     secrets:
       OSSRH_USER: ${{ secrets.OSSRH_USER }}
       OSSRH_SECRET: ${{ secrets.OSSRH_SECRET }}
-      OSSRH_URL: ${{ secrets.OSSRH_SNAPSHOT_URL }}
+      OSSRH_URL: ${{ secrets.RELEASE_URL }}
```

Comment:
it should be OSSRH_URL: ${{ secrets.OSSRH_SNAPSHOT_URL }} only

Reply:
made change here --> https://github.com/mosip/digital-credential-plugins/pull/61

Reply:
@Prafulrakhade @praful-technoforte I've made the above suggested changes, but still the maven builds are failing, can you please take a look at this?